packages/email-notification-dispatcher/Dockerfile

```
@@ -1,7 +1,7 @@
 #default value for compatibility
 ARG NODE_REGISTRY="docker.io"
 
-FROM ${NODE_REGISTRY}/node:20.19.3-slim@sha256:f8f6771d949ff351c061de64ef9cbfbc5949015883fb3b016b34aca6d0e5b8cc as build 
+FROM ${NODE_REGISTRY}/node:20.19.4-slim@sha256:ef358d9fa0734e2c3b42482aa352c28a59245bc3d32e07ef57810cf8ba9092c5 AS build
 
 RUN npm install -g corepack@0.31.0 && corepack enable
```

```suggestion
RUN corepack enable
```
Corepack is already present: The node:20.19.4-slim image already includes the default version of Corepack